src/features/Configuration/boardDefinitions.ts

```diff
@@ -86,7 +89,13 @@ export function getBoardDefinition(
 
         case 'PCA10175':
             // nRF54H20
-            return { boardControllerConfigDefinition: typednrf54h20v070json };
+            if (boardRevision === '0.7.0') {
```

Comment:
I suspect there might be more versions than 0.7.0. We should have a look around to see which DPKs are available. (Otherwise the new file will be default for all otheer H20s (which _might_ be fine, altogether)

Reply:
True. Thought that is something to check closer to the date when merge is required